server/db/migrations/20170420000209-addForeignKeys_to_PatientProvider.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    const addPatient = queryInterface.addColumn('PatientProviders', 'Patient_id', {
+      type: Sequelize.INTEGER,
+      references: {
+        model: 'Patients', // NEED THE STUPID 's' c'mon!!
```

Comment:
@alexrosenkranz  ... Dude this bug took me soo long to figure out! I forgot that sequelize automatically pluralizes your model names if you don't freeze the name, so I was getting a fricken error of not being able to add foreign keys etc. .... fun stuff.